### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by elizabethkim11**
Footer w/ social icons + corresponding links

### CODE REVIEW

**Comment by nendow02** [comment]
Fixed the default mail icon issue, more info on the CSS I used can be found [here](https://developer.mozilla.org/en-US/docs/Web/CSS/::before). Lmk if you have any questions on what changed. I’ll review your PR rn.

File: styles/Home.module.scss [hunk 1]
```diff
@@ -14,13 +14,30 @@
   align-items: center;
 }
 
-.footer {
```
Comment (nendow02) [inline styles/Home.module.scss:17]: Can we put all of your footer css in its own module?

File: styles/Home.module.scss [hunk 2]
```diff
@@ -14,13 +14,30 @@
...
-  padding: 2rem 0;
+  padding: 1rem 5;
   border-top: 1px solid #eaeaea;
   justify-content: center;
   align-items: center;
+  font-weight: bold;
+}
+
+.footerlogo {
+  margin-top: 03rem;
```
Comment (nendow02) [inline styles/Home.module.scss]: Can we remove margin-top here to make footer slightly smaller

File: styles/Home.module.scss [hunk 3]
```diff
@@ -14,13 +14,30 @@
...
+}
+
+.mailanchor::before {
+  content: "" !important;
+}
+
+.copyright {
+  display: flex;
+  flex: 1;
+  padding: 1rem 2rem;
```
Comment (nendow02) [inline styles/Home.module.scss]: Can we also remove top padding here (keep all others)

**Review by nendow02 — CHANGES_REQUESTED** [review]
Nice job! Few requested changes one of which is moving css for footer into its own file. For the style changes those are more of suggestions, this is your ticket so if you have other ideas on these requests feel free to comment that!

**Review by andyren12 — APPROVED** [review]
looks good

**Review by nendow02 — APPROVED** [review]
Go ahead and squash and merge!

**Pull request merged**